pom.xml

```diff
@@ -106,7 +107,7 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
             <version>2.0.9</version>
-            <scope>test</scope>
+<!--            <scope>test</scope>-->
```

Comment:
why is this commented out? slf4j-simple is not intended to be a transitive dependency

Reply:
Hmm my fault - do not remember exactly the purpose of doing it..